src/main/resources/messages/messages.properties

```diff
@@ -767,6 +778,9 @@ message.spinner.build_has_failed=The build has failed
 message.spinner.uploading_strings=Uploading strings
 message.spinner.uploading_strings_percents=Uploading strings (%d%%)
 message.spinner.upload_strings_failed=Upload has failed
+message.spinner.cloning_branch=Cloning branch
+message.spinner.cloning_branch_percents=Cloning branch @|bold (%d%%)|@
+message.spinner.clone_failed=Clone failed
```

Comment:
We already have a similar message above - `error.branch.clone`. Can we use a single message instead of keeping two almost identical messages?